Modules/Core/Common/include/itkThreadSupport.h

```diff
@@ -52,7 +52,7 @@ using ThreadProcessIdType = pthread_t;
 constexpr ThreadProcessIdType ITK_DEFAULT_THREAD_ID = 0;
 using ITK_THREAD_RETURN_TYPE = void *;
 constexpr ITK_THREAD_RETURN_TYPE ITK_THREAD_RETURN_DEFAULT_VALUE =
-  NULL;                                     /* This is from a c library, and always needs to be NULL, not nullptr */
+  nullptr;                                  /* This is from a c library, and always needs to be NULL, not nullptr */
```

Comment:
And these few `NULL`s have a big warning next to them about not changing them. Written by you, no less 😈